include/engine/phantom_node.hpp

```diff
@@ -177,7 +186,7 @@ struct PhantomNode
 };
 
 #ifndef _MSC_VER
-static_assert(sizeof(PhantomNode) == 52, "PhantomNode has more padding then expected");
+static_assert(sizeof(PhantomNode) == 60, "PhantomNode has more padding then expected");
```

Comment:
Will look into this, but it might take a while, as I have to get a build going first.
Serach for `: error` in https://ci.appveyor.com/project/DennisOSRM/osrm-backend/build/1.0.2784
e.g. have to adjust [`windows-builds`](https://github.com/mapbox/windows-builds) to build  `boost_unit_test_framework` and create a new binary deps package.